tests/functional/adapter/materialization/test_table_drop.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+from dbt.tests.util import run_dbt
+
+from tests.functional.adapter.materialization.fixtures import (
+    table_drop_csv,
+    table_drop_sql,
+    table_drop_yml,
+)
+
+
+class TestTableDrop:
```

Comment:
I migrated `on_table_exists = drop` tests to the new testing adapter framework. Other tests from `docker/dbt/jaffle_shop` can be migrated later on since some from those tests could be inherited from `dbt-core` or `dbt-tests-adapter` once dbt-labs team move them there.

I will create a follow up issue to migrate outstanding tests from `docker/dbt/jaffle_shop`.